src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/GatherTranches.java

```diff
@@ -0,0 +1,87 @@
+package org.broadinstitute.hellbender.tools.walkers.vqsr;
+
+import htsjdk.samtools.util.IOUtil;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.ReadProgramGroup;
+import org.broadinstitute.hellbender.exceptions.UserException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.*;
+
+@CommandLineProgramProperties(
+        summary = "Gathers scattered VQSLOD tranches into a single file",
+        oneLineSummary = "Gathers scattered VQSLOD tranches into a single file",
+        programGroup = ReadProgramGroup.class
+)
+@DocumentedFeature
+public class GatherTranches extends CommandLineProgram {
+    @Argument(fullName = StandardArgumentDefinitions.INPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME, doc="List of scattered tranches files")
```

Comment:
Can you assert that at least one should be provided?  Is optional=true enough to do that?